radar/radar.go

```diff
@@ -68,7 +68,17 @@ func NewRadar(
 
 func (radar *Radar) Scanner(logger lager.Logger, resourceName string) ifrit.Runner {
 	return ifrit.RunFunc(func(signals <-chan os.Signal, ready chan<- struct{}) error {
-		ticker := time.NewTicker(radar.interval)
+		var ticker *time.Ticker
+		resourceConfig, err := radar.getResourceConfig(logger, resourceName)
```

Comment:
I could get behind that though it seems a bit weird that scan now has this interval it returns.  Would you envisage that Radar has the timer directly on itself?